build/Makefile

```diff
@@ -296,6 +296,8 @@ $(INTERMEDIATE_DIR)/$(OUTPUT_PACKAGE_PREFIX).tar : $(RUBY_DEST_DIR) $(IN_PLUGINS
 	cd $(DSC_DIR)/release; $(COPY) `ls omsconfig-*.ssl_100.*.deb | sort | tail -1` $(INTERMEDIATE_DIR)/100
 	cd $(INTERMEDIATE_DIR)/100; for f in omsconfig-*.{rpm,deb}; do SOURCE=$$f; DEST=`echo $$SOURCE | sed 's/.ssl_100././'` ; $(MV) $$SOURCE $$DEST; done
 
+	cd $(DSC_DIR)/release; $(COPY) `ls *.zip` $(TARGET_DIR)/
```

Comment:
Sorry for the delay in the detailed review.

While, technically, your change would work, I wouldn't suggest this. The area you're putting the change into is copying bits for the embedded tar file in the shell bundle, and this has nothing to do with that.

At line 257, there is a dependency for building a kit. I'd make this part of that. The current line at 257 reads:

```
kit : $(TARGET_DIR)/$(OUTPUT_PACKAGE_PREFIX).sh
```

I would replace this with something like:

```
kit : $(TARGET_DIR)/$(OUTPUT_PACKAGE_PREFIX).sh $(TARGET_DIR)/dsc

$(TARGET_DIR)/dsc : $(DSC_TARGET_DIR)
	@echo "========================= Performing Copying DSC resources"
	cd $(DSC_TARGET_DIR); $(COPY) *.zip $(TARGET_DIR)/ 
```

This puts the change in it's own section, where it belongs since it has nothing to do with building a .tar file. It's also dependency based, insuring that DSC is built first due to prior rules.

Finally, there was no need to do an `ls *.tar` in your copy line. You can just copy directly without doing the 'ls' operation.

Run this through pbuild to insure that it does what we expect. But this looks right.